filesystem/apm_partition_table.ksy

```diff
@@ -0,0 +1,69 @@
+meta:
+  id: apm_partition_table
+  title: APM (Apple Partition Map) partition table
+  license: CC0-1.0
+  endian: be
+  encoding: ascii
```

Comment:
I'm not 100% sure that it would be ASCII and not just some raw 8-bit system-dependent encoding. Any hints?

Reply:
From Apple's rescue CD:
"Apple", "Apple_partition_map", "Macintosh", "Apple_driver43"

It all looks ASCII to me, I think only IBM is old enough to pre-date ASCII with EBCDIC

Reply:
The problem is that real ASCII is 7-bit. Occasionally, people refer to something as ASCII, but actually it turns out that there's some (usually system config dependent) 8-bit encoding like cp437 or whatever's old Mac equivalent. Trying to parse that as "ASCII" would fail with a runtime error.

Reply:
The document quotes it as "fixed ASCII right-side NULL padded"
I have not found the reference of my reference.